pkg/constants/constants.go

```diff
@@ -246,6 +248,8 @@ var (
 	IstioMeshGateway = "mesh"
 )
 
+const WorkerNodePostfix = "worker"
```

Comment:
what does postfix stand for in this case?

Reply:
`WorkerNodePostfix` is used to clearly identify worker nodes by appending it to other strings.

Reply:
`Postfix` name is confusing

Reply:
I changed it to WorkerNodeName like KServeName. What about this?

Reply:
I think you want "suffix"?

Reply:
let me know what would be the best :) current version is `WorkerNodeName`

Reply:
ok it is now `WorkerNodeSuffix`